asm/f64/asm_test.go

```diff
@@ -69,6 +69,12 @@ func nonStridedWrite(x []float64, inc int) bool {
 	return false
 }
 
+var ( // Offset sets for testing alignment handling in Unitary assembly functions.
+	align2 = []struct{ x, y int }{{0, 0}, {0, 1}, {1, 0}, {1, 1}}
```

Comment:
```
align2 = newIncSet(0, 1)
align3 = newIncToSe(0, 1)
```

Also, squash `thiscommit` with `thiscommit~1` to make reason for existence clear.